framework/src/mesh/MooseMesh.C

```diff
@@ -1233,9 +1233,12 @@ MooseMesh::detectOrthogonalDimRanges(Real tol)
     }
   }
 
+  this->comm().max(max);
+  this->comm().min(min);
+
   _extreme_nodes.resize(8); // 2^LIBMESH_DIM
   // Now make sure that there are actual nodes at all of the extremes
-  unsigned int extreme_matches = 0;
+  std::vector<bool> extreme_matches(8, false);
```

Comment:
No, this is fine, just saw the fixed size and had my knee-jerk reaction. I'm fine with vector<bool> in non-threaded regions.